flink-core/src/main/java/org/apache/flink/configuration/AkkaOptions.java

```diff
@@ -42,22 +45,27 @@
 	public static final ConfigOption<String> WATCH_HEARTBEAT_INTERVAL = ConfigOptions
 		.key("akka.watch.heartbeat.interval")
 		.defaultValue(ASK_TIMEOUT.defaultValue())
-		.withDescription("Heartbeat interval for Akka’s DeathWatch mechanism to detect dead TaskManagers. If" +
-			" TaskManagers are wrongly marked dead because of lost or delayed heartbeat messages, then you should" +
-			" decrease this value or increase akka.watch.heartbeat.pause. A thorough description of Akka’s DeathWatch" +
-			" can be found <a href=\"http://doc.akka.io/docs/akka/snapshot/scala/remoting.html#failure-detector\">here</a>.");
+		.withDescription(Description.builder()
+			.text("Heartbeat interval for Akka’s DeathWatch mechanism to detect dead TaskManagers. If" +
```

Comment:
we could think about adding another version of `withDescription` that works like `text`, so that you could write this as
```
withDescription(
	"Heartbeat interval for Akka’s DeathWatch mechanism to detect dead TaskManagers. If" +
		..
		" Akka’s DeathWatch can be found %s",
	link("http://doc.akka.io/docs/akka/snapshot/scala/remoting.html#failure-detector", "here"));
```

Just a thought.